### ISSUE TITLE
Deprecated .egg files when running the install command

### ISSUE THREAD

**Issue by RunT00** [PrithivirajDamodaran/Parrot#59]
**when running** `>!pip install git+https://github.com/PrithivirajDamodaran/Parrot_Paraphraser.git`**, the output for this part is:**

Collecting git+https://github.com/PrithivirajDamodaran/Parrot_Paraphraser.git
  Cloning https://github.com/PrithivirajDamodaran/Parrot_Paraphraser.git to c:\users\32067\appdata\local\temp\pip-req-build-aaa3h5n1
  Resolved https://github.com/PrithivirajDamodaran/Parrot_Paraphraser.git to commit 720a87a1ee557d8ed8d9a021adbdd1dd5616c5f9
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'done'
Requirement already satisfied: transformers in c:\programdata\anaconda3\lib\site-packages (from parrot==1.0) (4.32.1)
Requirement already satisfied: sentencepiece in c:\programdata\anaconda3\lib\site-packages\sentencepiece-0.1.99-py3.11-win-amd64.egg (from parrot==1.0) (0.1.99)
Requirement already satisfied: python-Levenshtein in c:\programdata\anaconda3\lib\site-packages\python_levenshtein-0.23.0-py3.11.egg (from parrot==1.0) (0.23.0)
Requirement already satisfied: sentence-transformers in c:\programdata\anaconda3\lib\site-packages\sentence_transformers-2.2.2-py3.11.egg (from parrot==1.0) (2.2.2)
Requirement already satisfied: fuzzywuzzy in c:\programdata\anaconda3\lib\site-packages\fuzzywuzzy-0.18.0-py3.11.egg (from parrot==1.0) (0.18.0)
Requirement already satisfied: Levenshtein==0.23.0 in c:\programdata\anaconda3\lib\site-packages\levenshtein-0.23.0-py3.11-win-amd64.egg (from python-Levenshtein->parrot==1.0) (0.23.0)
Requirement already satisfied: rapidfuzz<4.0.0,>=3.1.0 in c:\programdata\anaconda3\lib\site-packages\rapidfuzz-3.6.1-py3.11-win-amd64.egg (from Levenshtein==0.23.0->python-Levenshtein->parrot==1.0) (3.6.1)
Requirement already satisfied: tqdm in c:\programdata\anaconda3\lib\site-packages (from sentence-transformers->parrot==1.0) (4.65.0)
Requirement already satisfied: torch>=1.6.0 in c:\programdata\anaconda3\lib\site-packages\torch-2.1.2-py3.11-win-amd64.egg (from sentence-transformers->parrot==1.0) (2.1.2)
Requirement already satisfied: torchvision in c:\programdata\anaconda3\lib\site-packages\torchvision-0.16.2-py3.11-win-amd64.egg (from sentence-transformers->parrot==1.0) (0.16.2)
Requirement already satisfied: numpy in c:\programdata\anaconda3\lib\site-packages (from sentence-transformers->parrot==1.0) (1.24.3)
Requirement already satisfied: scikit-learn in c:\programdata\anaconda3\lib\site-packages (from sentence-transformers->parrot==1.0) (1.3.0)
Requirement already satisfied: scipy in c:\programdata\anaconda3\lib\site-packages (from sentence-transformers->parrot==1.0) (1.11.1)
Requirement already satisfied: nltk in c:\programdata\anaconda3\lib\site-packages (from sentence-transformers->parrot==1.0) (3.8.1)
Requirement already satisfied: huggingface-hub>=0.4.0 in c:\programdata\anaconda3\lib\site-packages (from sentence-transformers->parrot==1.0) (0.15.1)
Requirement already satisfied: filelock in c:\programdata\anaconda3\lib\site-packages (from transformers->parrot==1.0) (3.9.0)
Requirement already satisfied: packaging>=20.0 in c:\programdata\anaconda3\lib\site-packages (from transformers->parrot==1.0) (23.1)
Requirement already satisfied: pyyaml>=5.1 in c:\programdata\anaconda3\lib\site-packages (from transformers->parrot==1.0) (6.0)
Requirement already satisfied: regex!=2019.12.17 in c:\programdata\anaconda3\lib\site-packages (from transformers->parrot==1.0) (2022.7.9)
Requirement already satisfied: requests in c:\programdata\anaconda3\lib\site-packages (from transformers->parrot==1.0) (2.31.0)
Requirement already satisfied: tokenizers!=0.11.3,<0.14,>=0.11.1 in c:\programdata\anaconda3\lib\site-packages (from transformers->parrot==1.0) (0.13.2)
Requirement already satisfied: safetensors>=0.3.1 in c:\programdata\anaconda3\lib\site-packages (from transformers->parrot==1.0) (0.3.2)
Requirement already satisfied: fsspec in c:\programdata\anaconda3\lib\site-packages (from huggingface-hub>=0.4.0->sentence-transformers->parrot==1.0) (2023.4.0)
Requirement already satisfied: typing-extensions>=3.7.4.3 in c:\programdata\anaconda3\lib\site-packages (from huggingface-hub>=0.4.0->sentence-transformers->parrot==1.0) (4.7.1)
Requirement already satisfied: sympy in c:\programdata\anaconda3\lib\site-packages (from torch>=1.6.0->sentence-transformers->parrot==1.0) (1.11.1)
Requirement already satisfied: networkx in c:\programdata\anaconda3\lib\site-packages (from torch>=1.6.0->sentence-transformers->parrot==1.0) (3.1)
Requirement already satisfied: jinja2 in c:\programdata\anaconda3\lib\site-packages (from torch>=1.6.0->sentence-transformers->parrot==1.0) (3.1.2)
Requirement already satisfied: colorama in c:\programdata\anaconda3\lib\site-packages (from tqdm->sentence-transformers->parrot==1.0) (0.4.6)
Requirement already satisfied: click in c:\programdata\anaconda3\lib\site-packages (from nltk->sentence-transformers->parrot==1.0) (8.0.4)
Requirement already satisfied: joblib in c:\programdata\anaconda3\lib\site-packages (from nltk->sentence-transformers->parrot==1.0) (1.2.0)
Requirement already satisfied: charset-normalizer<4,>=2 in c:\programdata\anaconda3\lib\site-packages (from requests->transformers->parrot==1.0) (2.0.4)
Requirement already satisfied: idna<4,>=2.5 in c:\programdata\anaconda3\lib\site-packages (from requests->transformers->parrot==1.0) (3.4)
Requirement already satisfied: urllib3<3,>=1.21.1 in c:\programdata\anaconda3\lib\site-packages (from requests->transformers->parrot==1.0) (1.26.16)
Requirement already satisfied: certifi>=2017.4.17 in c:\programdata\anaconda3\lib\site-packages (from requests->transformers->parrot==1.0) (2023.11.17)
Requirement already satisfied: threadpoolctl>=2.0.0 in c:\programdata\anaconda3\lib\site-packages (from scikit-learn->sentence-transformers->parrot==1.0) (2.2.0)
Requirement already satisfied: pillow!=8.3.*,>=5.3.0 in c:\programdata\anaconda3\lib\site-packages (from torchvision->sentence-transformers->parrot==1.0) (10.0.1)
Requirement already satisfied: MarkupSafe>=2.0 in c:\programdata\anaconda3\lib\site-packages (from jinja2->torch>=1.6.0->sentence-transformers->parrot==1.0) (2.1.1)
Requirement already satisfied: mpmath>=0.19 in c:\programdata\anaconda3\lib\site-packages (from sympy->torch>=1.6.0->sentence-transformers->parrot==1.0) (1.3.0)
**DEPRECATION: Loading egg at c:\programdata\anaconda3\lib\site-packages\fuzzywuzzy-0.18.0-py3.11.egg is deprecated. pip 23.3 will enforce this behaviour change. A possible replacement is to use pip for package installation..
DEPRECATION: Loading egg at c:\programdata\anaconda3\lib\site-packages\levenshtein-0.23.0-py3.11-win-amd64.egg is deprecated. pip 23.3 will enforce this behaviour change. A possible replacement is to use pip for package installation..
DEPRECATION: Loading egg at c:\programdata\anaconda3\lib\site-packages\python_levenshtein-0.23.0-py3.11.egg is deprecated. pip 23.3 will enforce this behaviour change. A possible replacement is to use pip for package installation..
DEPRECATION: Loading egg at c:\programdata\anaconda3\lib\site-packages\rapidfuzz-3.6.1-py3.11-win-amd64.egg is deprecated. pip 23.3 will enforce this behaviour change. A possible replacement is to use pip for package installation..
DEPRECATION: Loading egg at c:\programdata\anaconda3\lib\site-packages\sentencepiece-0.1.99-py3.11-win-amd64.egg is deprecated. pip 23.3 will enforce this behaviour change. A possible replacement is to use pip for package installation..
DEPRECATION: Loading egg at c:\programdata\anaconda3\lib\site-packages\sentence_transformers-2.2.2-py3.11.egg is deprecated. pip 23.3 will enforce this behaviour change. A possible replacement is to use pip for package installation..
DEPRECATION: Loading egg at c:\programdata\anaconda3\lib\site-packages\torch-2.1.2-py3.11-win-amd64.egg is deprecated. pip 23.3 will enforce this behaviour change. A possible replacement is to use pip for package installation..
DEPRECATION: Loading egg at c:\programdata\anaconda3\lib\site-packages\torchvision-0.16.2-py3.11-win-amd64.egg is deprecated. pip 23.3 will enforce this behaviour change. A possible replacement is to use pip for package installation..**
  Running command git clone --filter=blob:none --quiet https://github.com/PrithivirajDamodaran/Parrot_Paraphraser.git 'C:\Users\32067\AppData\Local\Temp\pip-req-build-aaa3h5n1'

**No errors occurred when executing the whole program, but the final outputs are always "No paraphrases returned" even if I have lowered down the thresholds. 
I have already installed these packages, but it seems like there is a problem with them.**

**Comment by RunT00** [comment]
I have reinstalled PyTorch in Conda environment and got the correct ouputs.
